scripts/pipe-convert-for-readme

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# To use this script, pipe the contents of a MD file into it.
+# It will output that content reformatted for Readme.
+
+# Examples:
+# Reformat the contents of the clipboard
+# xclip -o | ./pipe-convert-for-readme | xclip -selection clipboard
+#
+# Reformat a file and write it to a new file
+# cat ../_includes/devdoc/guide_block_chain.md | ./pipe-convert-for-readme > ../_readme/guide_block_chain.md
+
+perl -0777 \
+-pe "s/\{% comment %\}[^{]+\{% endcomment %\}//gs;" \
+-pe "s/\[([^]]+)\]\[.+?\.term\}/<<glossary:\1>>/g;" \
+-pe "s/<<([^>]+)\n([^>]+)>>/<<\1 \2>>/gs;" \
+-pe "s!/img/dev!https://dash-docs.github.io/img/dev!g;" \
+-pe "s/{[^}]+}//g;" \
```

Comment:
Also, this guy is clobbering parts of code samples and JSON results :frowning:

Reply:
I incorporated the image correction in the latest version. I also remove HTML comments now, and soon this will reformat the NTPD tables.